feat: adding recipes tracker workshop (#56456)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-recipe-tracker/670e4f45f7116c0f216a5177.md

---
id: 670e4f45f7116c0f216a5177
title: Step 5
challengeType: 1
dashedName: step-5
---

# --description--

Before we move on, you should practice how to access properties from an object.

> For this, you can use a dot (`.`) or a bracket notation (`[]`). Here's how to do that:

Reviewer: ```suggestion
You can use either dot (`.`) or bracket (`[]`) notation to do this. Here's an example:
```

```js
const person = {
  name: 'John',
  age: 30,
  job: 'Software Engineer'
};

console.log(person.name); // John
console.log(person['age']);  // 30
```

> Access the `name` properties of both `recipe1` and `recipe2`. Set them to the variables `recipe1Name` and `recipe2Name` respectively.

Reviewer: ```suggestion
Access the `name` properties of both `recipe1` and `recipe2`, and assign them to the variables `recipe1Name` and `recipe2Name`, respectively.
```

> Also access the `cookingTime` of both recipes. Set them to the variables `recipe1CooklingTime` and `recipe2CooklingTime` respectively.

Reviewer: ```suggestion
Next, access the `cookingTime` properties of both recipes and assign them to the variables `recipe1CookingTime` and `recipe2CookingTime`, respectively.
```
